dags/common.py

```diff
@@ -0,0 +1,23 @@
+import dagster
+
+from posthog.clickhouse.cluster import (
+    ClickhouseCluster,
+    get_cluster,
+)
+
+
+class ClickhouseClusterResource(dagster.ConfigurableResource):
+    """
+    The ClickHouse cluster used to run the job.
+    """
+
+    client_settings: dict[str, str] = {
+        "lightweight_deletes_sync": "0",
+        "max_execution_time": "0",
+        "max_memory_usage": "0",
+        "mutations_sync": "0",
```

Comment:
Could make a reasonable argument that the `*_sync` settings should just be provided by the mutation runner instead so that the behavior is more explicit/obvious at the call site but either way works